knexfile.js

```diff
@@ -1,5 +1,13 @@
 const dotenvSafe = require('dotenv');
 dotenvSafe.config(); // No dotenv-safe otherwise tests will fail due to settings at runtime
+const toCamelCase = require('camelcase-keys');
+const _ = require('lodash');
```

Comment:
I prever `const { forEach, set, snakeCase } = require('lodash')` but the choice is yours 😉